README.md

```diff
@@ -39,7 +39,7 @@ microcontrollers).
 
 ## Minimum Supported Rust Version
 
-This crate requires **Rust 1.83** at a minimum.
+This crate requires **Rust 1.85** at a minimum.
```

Comment:
You also need to update the badge. And I think we can remove this section completely. With the new MSRV-aware resolver we can bump MSRV without issues even in patch releases.